code/modules/chemistry/Chemistry-Holder.dm

```diff
@@ -59,6 +59,8 @@ datum
 
 		var/postfoam = 0 //attempt at killing infinite foam
 
+		var/gnesisstasis = 0//used for pausing gnesis for stuff
```

Comment:
I don't quite like this way of doing this. Couldn't it be a var on the gnesis reagent itself?